### PR TITLE
Feature/display

### PR DESCRIPTION
**Pull request by niehaus-alexander**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+  align-items: center;
+  margin: 0;
+  height: 50vh;
+`;
+
+const StyledListItem = styled.li`
+  border: 2px black solid;
+  border-radius: 5px;
+  padding: 5px;
+  cursor: pointer;
```
Comment (doemser) [inline pages/index.js]: why is this indicating to be clickable, a listItem should never clickable. We only click on buttons or links, in general

File: styles.js [hunk 2]
```diff
@@ -11,4 +11,13 @@ export default createGlobalStyle`
...
   }
+
+ 
+
+ 
+  
+  
+
+
+
```
Comment (doemser) [inline styles.js]: Do you need this extra 8 empty lines of css?

**Review by doemser — CHANGES_REQUESTED** [review]
Good first one, just remove the indication that the lititems are clickable.

**Review by Braun-Johannes — APPROVED** [review]
LGTM

**Pull request merged**